src/pages/Taxonomy/index.js

```diff
@@ -747,39 +751,63 @@ const getURLFromState = createSelector(
   (state) => state.customLocation.description,
   (state) => state.customLocation.search,
   ({ protocol, hostname, port, root }, description, { search }) => {
-    const desc = {
-      ...description,
-      taxonomy: {
-        db: 'uniprot',
-        accession: search,
-      },
-    };
-    try {
-      return format({
-        protocol,
-        hostname,
-        port,
-        pathname: root + descriptionToPath(desc),
-      });
-    } catch {
-      return;
+    if (search && search.match(/^\d+$/) && description.taxonomy) {
+      const desc = {
+        ...description,
+        taxonomy: {
+          db: 'uniprot',
+          accession: search,
+        },
+      };
+      try {
+        return format({
+          protocol,
+          hostname,
+          port,
+          pathname: root + descriptionToPath(desc),
+        });
+      } catch {
+        return;
+      }
+    } else if (search && search.match(/^[\w ]+$/) && description.taxonomy) {
+      const desc = {
+        main: {
```

Comment:
Not having `...description` here, makes the query to lose any other endpoints that are been filtered by. So the `scientific_name` searches over all the taxonomy names.
This might not be a bad thing because it allows to show inner tax nodes.

But it causes undesired side effects, like the counters will include more entries/proteins.

For example, check the human entry here:
http://localhost:8000/interpro/taxonomy/uniprot/entry/smart/?search=sapiens#table
v.s.
http://localhost:8000/interpro/taxonomy/uniprot/entry/smart/?search=homo%20sapiens#table

The problem is that it seems that the API is not supporting the `scientific_name` search for multiple endpoints, as for example this query fails:
http://localhost:8007/api//taxonomy/uniprot/entry/interpro?scientific_name=Homo%20sapiens